tensorflow/compiler/xla/service/depthwise_convolution_converter.h

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 #define TENSORFLOW_COMPILER_XLA_SERVICE_DEPTHWISE_CONVOLUTION_CONVERTER_H_
 
 #include "absl/strings/string_view.h"
+#include <functional>
```

Comment:
nit: this stl include should be in its own block above #include "absl/strings/string_view.h"
So:
```
#include <functional>

#include "absl/strings/string_view.h"
```